x/escrow/keeper/keeper.go

```diff
@@ -3,6 +3,8 @@ package keeper
 import (
 	"github.com/cosmos/cosmos-sdk/codec"
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	bankTypes "github.com/cosmos/cosmos-sdk/x/bank/types"
```

Comment:
When you have to give a name to an import, we generally use all lowercase. So in this case it would be `banktypes`

Reply:
Done